mongodbatlas/resource_mongodbatlas_cluster_test.go

```diff
@@ -365,6 +365,37 @@ func TestAccResourceMongoDBAtlasCluster_AWSWithLabels(t *testing.T) {
 	})
 }
 
+func TestAccResourceMongoDBAtlasClusterPrivateEndpointLink_(t *testing.T) {
```

Comment:
Could you change the name of the function maybe it could be `TestAccResourceMongoDBAtlasCluster_withPrivateEndpointLink`

Reply:
sure, no problem